assemble/bin/accumulo-cluster

```diff
@@ -156,9 +164,9 @@ function control_service() {
     else
       if [[ $# -gt 3 ]]; then
         EXTRA_ARGS="${*:4}"
-        $SSH "$host" "bash -c 'ACCUMULO_SERVICE_INSTANCE=${ACCUMULO_SERVICE_INSTANCE} ${bin}/accumulo-service \"$service\" \"$control_cmd\" \"-a\" \"$host\" $EXTRA_ARGS '"
+        $SSH "$host" "bash -c 'ACCUMULO_SERVICE_INSTANCE=${ACCUMULO_SERVICE_INSTANCE} ${bin}/accumulo-service \"$service\" \"$control_cmd\" \"-a\" \"$host\" $EXTRA_ARGS '&"
       else
-        $SSH "$host" "bash -c 'ACCUMULO_SERVICE_INSTANCE=${ACCUMULO_SERVICE_INSTANCE} ${bin}/accumulo-service \"$service\" \"$control_cmd\" \"-a\" \"$host\"'"
+        $SSH "$host" "bash -c 'ACCUMULO_SERVICE_INSTANCE=${ACCUMULO_SERVICE_INSTANCE} ${bin}/accumulo-service \"$service\" \"$control_cmd\" \"-a\" \"$host\"'&"
```

Comment:
I'm not so sure about having `&` on the end of this. If the command is backgrounded, it might close the SSH connection, which will terminate the command.

Reply:
This was part of our speed up during scale testing when running multiple processes per host 
I'm happy to pull it out since it was a quick & easy fix and we can revisit it in 2.1.4

Reply:
Well, I definitely understand it being faster... I just don't trust it to not kill the command before it's finished starting things up. If you're sure it's always working... it's probably fine. Can keep it and just try to fix it later if it's buggy. I don't care that much. I still see these scripts as more of a minimal reference implementation of a cluster management utility. I'd be somewhat interested in seeing us just use pssh instead, to try to get the parallelism without the risks.

Reply:
I can pull it just so we don't have a buggy version of 2.1.3. 
It's an easy enough modification to make at a later date.

Reply:
and agreed, using something like pssh or pdsh would be simpler